var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -37,11 +37,16 @@ class Pango(Package):
 
     version('1.36.8', '217a9a753006275215fa9fa127760ece')
     version('1.40.1', '6fc88c6529890d6c8e03074d57a3eceb')
+    version('1.40.3', '17c26720f5a862a12f7e1745e2f1d966')
```

Comment:
please add newer versions on top and also switch places of `1.40.1` and `1.36.8`.